doc/source/extras/pytest_plugin.rst

```diff
@@ -0,0 +1,6 @@
+.. _ref_pytest_plugin_docs:
+
+PyVista Pytest Plugin
```

Comment:
Super nitpicky, but I wonder about the casing of "Pytest". Looking at https://docs.pytest.org it seems to me that the project styles itself as "pytest":

![screenshot from docs.pytest.org with a bunch of lowercase "pytest"s, even outside a coding context, at the start of a sentence](https://user-images.githubusercontent.com/17914410/226066667-ce214343-45d3-4829-aa55-cf9abafa77e8.png)

Reply:
5896d170fc908f79cb1b77dc830288700d17c749

Reply:
We're using "title case", so I updated it in the body, but didn't update the title.

Reply:
We could add an exception just like we do for PyVista and others. I don't insist though.

Reply:
I most admit I was lazy, but not that lazy, and only googled it and saw this.  Their website proper consistently uses lower case.  I think we could use either here.

![Screenshot_20230317-201842](https://user-images.githubusercontent.com/39341281/226072662-497ca625-9862-49d6-9587-67b33359d95d.png)

Reply:
So should we switch or keep titlecase?

Reply:
I'm on the fence about this as well. Looking at their docs, they consistently do not capitalize it, even when it starts their sentence case titles:
https://docs.pytest.org/en/7.2.x/contents.html

Honestly, I think it looks weird if it's the only word not capitalized, it's going to look weird. I think an exception would be [gRPC](https://grpc.io/), where GRPC (all caps) or Grpc looks weird. In this case, I think we're fine with "as-is" since it doesn't "look weird".